doc/build-windows-mingw.md

```diff
@@ -0,0 +1,240 @@
+1. Prepare your build system.
+
+1.1 Install msys shell:
+http://sourceforge.net/projects/mingw/files/Installer/mingw-get-setup.exe/download
+From MinGW installation manager -> All packages -> MSYS
+mark the following for installation:
+
+msys-base-bin
+msys-autoconf-bin
+msys-automake-bin
+msys-libtool-bin
+
+then click on Installation -> Apply changes
+
+Make sure no mingw packages are checked for installation or present from a previous install. Only the above msys packages should be installed. Also make sure that msys-gcc and msys-w32api packages are not installed.
```

Comment:
Move this comment above the Installation -> Apply changes statement so people see this before hitting Apply changes.